tests/unit/modules/test_boto3_route53.py

```diff
@@ -0,0 +1,153 @@
+"""
+    Tests for salt.modules.boto3_route53
+"""
+
+
+import random
+import string
+
+import salt.loader
+import salt.modules.boto3_route53 as boto3_route53
+from salt.utils.versions import LooseVersion
+from tests.support.mixins import LoaderModuleMockMixin
+from tests.support.mock import MagicMock, patch
+from tests.support.unit import TestCase, skipIf
+
+try:
+    import boto3
+
+    HAS_BOTO3 = True
+except ImportError:
+    HAS_BOTO3 = False
+
+# the boto3_route53 module relies on the connect_to_region() method
+# which was added in boto 2.8.0
+# https://github.com/boto/boto/commit/33ac26b416fbb48a60602542b4ce15dcc7029f12
+REQUIRED_BOTO3_VERSION = "1.2.1"
+
+
+def __virtual__():
+    """
+    Returns True/False boolean depending on if Boto3 is installed and correct
+    version.
+    """
+    if not HAS_BOTO3:
+        return False
+    if LooseVersion(boto3.__version__) < LooseVersion(REQUIRED_BOTO3_VERSION):
+        return (
+            False,
+            "The boto3 module must be greater or equal to version {}".format(
+                REQUIRED_BOTO3_VERSION
+            ),
+        )
+    return True
+
+
+REGION = "us-east-1"
+ACCESS_KEY = "GKTADJGHEIQSXMKKRBJ08H"
+SECRET_KEY = "askdjghsdfjkghWupUjasdflkdfklgjsdfjajkghs"
+CONN_PARAMETERS = {
+    "region": REGION,
+    "key": ACCESS_KEY,
+    "keyid": SECRET_KEY,
+    "profile": {},
+}
+
+LIST_RESOURCE_RECORD_SETS_RETURN = {
+    "IsTruncated": True,
+    "MaxItems": "100",
+    "NextRecordName": "blog3.saltstack.furniture.",
+    "NextRecordType": "CNAME",
+    "ResourceRecordSets": [
+        {
+            "Name": "blog.saltstack.furniture.",
+            "ResourceRecords": [{"Value": "127.0.0.1"}],
+            "TTL": 60,
+            "Type": "A",
+        },
+        {
+            "Name": "blog2.saltstack.furniture.",
+            "ResourceRecords": [{"Value": "127.0.0.1"}],
+            "TTL": 60,
+            "Type": "A",
+        },
+    ],
+}
+
+
+@skipIf(HAS_BOTO3 is False, "The boto module must be installed.")
```

Comment:
**suggestion** can just use the module's HAS_BOTO3 - either `skipIf(boto3_route53.HAS_BOTO3...` or import HAS_BOTO3 from the module